x-pack/plugins/security_solution/cypress/screens/rule_details.ts

```diff
@@ -92,6 +95,10 @@ export const TIMELINE_TEMPLATE_DETAILS = 'Timeline template';
 
 export const TIMESTAMP_OVERRIDE_DETAILS = 'Timestamp override';
 
+export const TIMELINE_FIELD = (field: string) => {
```

Comment:
Do we have any standards/guidelines for when to use all caps vs when to use camel case, here? I had assumed that functions would use camelcase as in `getDetails` below.